data/rapid_config.yaml

```diff
@@ -2,8 +2,13 @@
 # Rapid configuration-  for customizing runtime behavior. 
 # The use case for this file is to enable/disable features without
 # needing to open/edit/rebuild the code. 
-rapid_config: 
-  # This setting styles any features in the map with a special shadow effect if 
-  # and only if they were originally presented to the user as AI features
-  style_fb_ai_roads:
-    enabled: false
+internal:
+  rapid_config: 
```

Comment:
This 'rapid_config' block is selected if we run an 'npm build-internal'. The other public one is selected if we just do an 'npm build'.

Reply:
What about adding default values for the various features in here? Right now they all appear to  default to `true` or `false`, perhaps, but you don't know  what  available as a developer without hunting through the code. It might be handy to list them here as initial values, and it's also lets us choose which features to have on by default.

Reply:
Ah, you mean default values for features, orthogonal to whether the 'internal' or 'public' block is selected?